laplace-essay-on-probabilities.md

```diff
@@ -214,57 +214,57 @@ the sum of the possibilities of each favorable case. Let us clarify
-second principle, we add the possibility 1/2 of *cross* on the first
-toss, to the possibility 1/4 of *pillar* happening on the first toss
-and *cross* on the second, we will have 3/4 for the probability
+compound events, *heads* on the first and second tosses, and *cross*
+on the first toss, *tails* on the second. Now if, conforming to the
+second principle, we add the possibility 1/2 of *heads* on the first
+toss, to the possibility 1/4 of *tails* happening on the first toss
+and *heads* on the second, we will have 3/4 for the probability
 sought, which accords with what we found on the supposition of playing
-both tosses. This supposition does not change in the least the outcome
-we bet on for this event: it serves merely to reduce the diverse cases
+both tosses. This supposition does not change in the least the situation
+of a bettor on this event: it serves merely to reduce the diverse cases
 to equally-possible ones.
 
 **3rd Principle.** One of the most important points of the theory of
-Probabilities, and the one that lends the most to illusions, is the
+Probabilities, and the one that lends itself the most to illusions, is the
 way the probabilites increase and decrease by their mutual
 combinations. If the events are independent of each other, the
 probability of their existing together is the product of their
 particular probabilities. Thus the probability of achieving an ace
 with a single die being a sixth, that of achieving two aces in rolling
-two dice at once is a thirty-sixth. In fact, each of the faces of the
+two dice at once is a thirty-sixth. Indeed, each of the faces of the
 one die being combinable with the six faces of the other, there are
 thirty-six equally possible cases, among which only one yields the two
-aces. Generally, the probability that a simple event in the same
-circumstances, will occur in sequence a given number of times, is
-equal to the probability of this simple event, raised to a power
+aces. Generally, the probability that a single event in the same
+circumstances occurs in sequence a given number of times is
+equal to the probability of this single event, raised to a power
 indicated by this number. As the successive powers of a fraction less
 than unity diminish without cease, an event that depends on a series
 of quite large probabilities can become extremely improbable. Suppose
 that a fact reach us by twenty witnesses, in the manner that the first
 transmit it to the second, the second to the third, and so on. Suppose
 further that the probability of each testimony be 9/10: then that of the
 fact resulting from the testimonies will be less than one
-eighth. One cannot better compare this diminution of probability than
-to the extinction of the light from objects, by the interposition of
-multiple pieces of glass; an insignificant number of pieces sufficing
+eighth. One cannot better compare this reduction of probability than
+to the extinction of the clarity of objects, by the interposition of
```

Comment:
I'm pretty sure this is about brightness rather than blurriness -- "clarity" strongly suggests the latter in English. I think I'll rephrase this using "brightness".

Reply:
I think it is about the blurriness that one observes when stacking glass (well, maybe not anymore ;), but admittedly, the two concepts aren't as distinct in French.

Reply:
s/clarity/brightness/ in the PR.